repository/src/main/java/tech/ydb/yoj/repository/db/StdTxManager.java

```diff
@@ -273,7 +255,7 @@ private static void checkSeparatePolicy(SeparatePolicy separatePolicy, String tx
             case ALLOW -> {
             }
             case STRICT ->
-                    throw new IllegalStateException(format("Transaction %s was run when another transaction is active", txName));
+                    throw new IllegalStateException("Transaction was run when another transaction is active");
```

Comment:
why remove the transaction name from the error message? It's still in the `LOG` switch case!